DbProvider.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect, createContext } from "react";
+import { Text, View } from "react-native";
+import * as SQLite from "expo-sqlite";
+
+export const DBProvider = createContext(undefined);
+
+export default function DataBaseProvider({ children }) {
```

Comment:
and please rename this provider to `DBProvider`, that way everything will be consistent